docs/website/blog/3.0.0-GA-blog.md

```diff
@@ -0,0 +1,333 @@
+---
+title: odo v3.0.0 GA Released 🎉
+author: Parthvi Vala
+author_url: https://github.com/valaparthvi
+author_image_url: https://github.com/valaparthvi.png
+tags: ["release"]
+slug: odo-v3-release
+---
+
+odo v3 is now GA! 🎉
+
+![](../static/img/logo_small.png)
+
+Are you an application developer wanting to work closely with a production like environment? But can't, because you either don't have the time or energy to dive in the steep learning curve involved in running your application on the cluster?
+
+`odo` is here to answer your problems!
```

Comment:
This bit was written because it was something on top of my head, but it can definitely use some editing.